actors/builtin/miner/monies.go

```diff
@@ -17,10 +18,18 @@ var LockTargetFactorNum = big.NewInt(3)
 var LockTargetFactorDenom = big.NewInt(10)
 
 // FF = (DeclaredFaultFactorNum / DeclaredFaultFactorDenom) * BR(t)
+// Filecoin Parameter: Amount of fee for faults that have been declared on time.
+// Motivation: This guarantees that a miner pays back the possible block reward earned since last deadline.
+// In current Filecoin, the network must be conservative and assume that the sector was faulty since the last time it was proven.
+// This penalty is currently overly punitive for continued faults.
 var DeclaredFaultFactorNum = big.NewInt(214)
 var DeclaredFaultFactorDenom = big.NewInt(100)
 
 // SP = (UndeclaredFaultFactor / DeclaredFaultFactorDenom) * BR(t)
+// Filecoin parameter: Amount of fee for faults that have not been declared on time.
+// Motivation: This fee is higher than FF for two reasons:
+// (1) it guarantees that a miner is incentivized to declare a fault early
+// (2) A miner stores less than (1-spacegap) of a sector, does not not declare it as faulty and hopes to get challenged on the stored parts. SP guarantees that on expectation this strategy would not earn positive rewards.
 var UndeclaredFaultFactorNum = big.NewInt(5)
 var UndeclaredFaultFactorDenom = big.NewInt(1)
```

Comment:
**Possible Change:** The fees of `FF` and `SP` are dependent on the soundness of the PoSt proof that we are using. This means that a future improvement for a PoSt for SDR (or a new PoSt for a new proof like NSE) could lead to a different soundness, which implicitly will lead to a different FF and SP.

We could have an overly punitive FF and SP and set the highest FF and SP as system wide parameters, however future optimization/proof changes could really minimize FF and SP.

The proposal here is to make FF and SP proof dependent - for example with NSE, SP could be drastically reduced (and made as close to FF) and also FF could be reduced.

Reply:
This is a significant change and would need careful thought, especially with regard to potential changed proof algorithms. We probably don't have time for it.

Reply:
moved to https://github.com/filecoin-project/specs-actors/issues/743